django_mongodb/features.py

```diff
@@ -342,12 +293,6 @@ def django_test_expected_failures(self):
             "delete_regress.tests.Ticket19102Tests.test_ticket_19102_select_related",
             "one_to_one.tests.OneToOneTests.test_o2o_primary_key_delete",
         },
-        "Cannot use QuerySet.update() when querying across multiple collections on MongoDB.": {
```

Comment:
I believe we should keep this error message. Raising "Using a QuerySet in annotate() is not supported on MongoDB" isn't helpful for the user since this isn't what they're doing but rather what Django is doing behind the scenes (kind of... really this error message needs to be changed since `Query` is used in more than just `annotate()`).

Reply:
Ok. yes, But the other test are going to be recategorized also?